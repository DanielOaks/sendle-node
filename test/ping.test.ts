import { client } from './helpers';

describe('Ping', () => {
  it('should return a ping', async () => {
    const { ping } = await client.ping();

    expect(ping).toBe('pong');
  });
});
