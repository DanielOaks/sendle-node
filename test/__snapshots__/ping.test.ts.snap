// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Ping > should return a ping 1`] = `"pong"`;
